e2e/utils/functions/odoo.ts

```diff
@@ -55,6 +64,66 @@ export class Odoo {
     await expect(this.page.locator('td.o_data_cell:nth-child(11)')).toHaveText('$ 16.00');
   }
 
+  async activateDeveloperMode() {
+    await this.navigateToSettings();
+    await expect(this.page.locator('#devel_tool a:nth-child(1)')).toBeVisible();
+    await this.page.locator('#devel_tool a:nth-child(1)').click();
+  }
+
+  async navigateToGroups() {
+    await this.navigateToSettings();
+    await expect(this.page.locator('ul.o_menu_sections>:nth-child(2)>a')).toBeVisible();
+    await this.page.locator('ul.o_menu_sections>:nth-child(2)>a').click();
+    await expect(this.page.getByRole('menuitem', { name: /groups/i })).toBeVisible();
+    await this.page.getByRole('menuitem', { name: /groups/i }).click();
+  }
+
+  async navigateToSettings() {
+    await this.page.locator("//a[contains(@class, 'full')]").click();
+    await expect(this.page.getByRole('menuitem', { name: /settings/i })).toBeVisible();
+    await this.page.getByRole('menuitem', { name: /settings/i }).click();
+  }
+
+  async createGroup() {
+    await this.page.getByRole('button', { name: /create/i }).click();
+    await this.page.getByLabel(/application/i).click();
+    await expect(this.page.getByText(/accounting/i)).toBeVisible();
+    await this.page.getByText(/accounting/i).click();
+    await this.page.getByLabel(/name/i).fill(`${randomOdooGroupName.groupName}`);
+    await this.page.getByRole('button', { name: /save/i }).click();
+    await delay(240000);
```

Comment:
That's a big delay?

Reply:
Manually, it takes 2 to 3 minutes for the sync to occur between Odoo and Keycloak. In the testing environment, it takes an additional minute for the test to pass.

Reply:
Yes, I remember the delay. Make sense!